### PR TITLE
[FIX] web_editor: reactivate automaticStep before returns

### PR DESCRIPTION
**Pull request by robinlej**
[Commit 1] made sure the history worked when resizing elements by
calling `odooEditor.automaticStepUnactive()`, but applied its
counterpart `automaticStepActive()` only at the very end of the action,
leaving some `return` statements on the way that could break the flow.

This commit calls `automaticStepActive` just before leaving the listener
and moves `automaticStepUnactive` just before the first DOM
modification. It's both more logical and avoids returns pitfalls.

Note: `automaticStepActive()` makes sure modifications made on the DOM
through the browser's developer tools are tracked and can be reversed
with the undo button. Not reactivating it in time means some flows could
be broken (until another method reactivates it).

[Commit 1]: https://github.com/odoo/odoo/commit/423f4bd2a6cc47e69699d2437eaa5acda94bb98d

Related to task-3576046

### CODE REVIEW

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
Add "Related to opw-XXX" in the commit message, that way we know where that came from even though not directly solving opw-XXX :+1:

File: addons/web_editor/static/src/js/editor/snippets.options.js [hunk 1]
```diff
@@ -4275,6 +4275,7 @@ registry.sizing = SnippetOptionWidget.extend({
 
             var resize = resizeValues[compass];
             if (!resize) {
+                self.options.wysiwyg.odooEditor.automaticStepActive("resizing");
```
[the remark this answers is not in the document]
Comment (robinlej) [reply addons/web_editor/static/src/js/editor/snippets.options.js]: @qsm-odoo You're right, that's updated.

**Comment by robinlej** [comment]
@qsm-odoo Good to go :+1:

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**